appserver/tests/appserv-tests/devtests/connector/v3/run_test.sh

```diff
@@ -51,9 +51,9 @@ test_run(){
   echo $ROOT
   cd $APS_HOME/../v2-tests/appserv-tests
```

Comment:
/v2-tests/appserv-tests folder would be deleted soon. That is the whole purpose of moving test from v2-tests/appserv-tests to appserv-tests . Please remove this reference